.rubocop_todo.yml

```diff
@@ -18,11 +18,11 @@ Lint/UnneededCopDisableDirective:
   Exclude:
     - 'app/controllers/application_controller.rb'
 
-# Offense count: 49
+# Offense count: 56
 Metrics/AbcSize:
   Max: 87
 
-# Offense count: 21
+# Offense count: 24
 # Configuration parameters: CountComments.
 Metrics/ClassLength:
```

Comment:
@colinbruce looks like this was the only cop that increased the max rather than excluding individual files so have readded the exclusions myself. The rest are new rails-specific cops.